apps/studio/components/interfaces/Integrations/CronJobs/CreateCronJobSheet.tsx

```diff
@@ -231,11 +238,6 @@ export const CreateCronJobSheet = ({
 
   const isEdited = form.formState.isDirty
 
-  // if the form hasn't been touched and the user clicked esc or the backdrop, close the sheet
-  if (!isEdited && isClosing) {
```

Comment:
@joshenlim this change causes a bad render of the "are you sure you want to close?". To replicate, try opening the sheet and clicking on the backdrop. 

Calling functions in render is ok if they don't cause a render loop https://react.dev/learn/you-might-not-need-an-effect#adjusting-some-state-when-a-prop-changes